### PR TITLE
hotbuild 1.0.7 (new formula)

### PR DESCRIPTION
**Pull request by wandercn**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by wandercn** [comment]
Do you need me to make any changes?

File: Formula/hotbuild.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  desc "Cross platfrom hot compilation tool for go"
+  homepage "https://hotbuild.ffactory.org"
+  url "https://github.com/wandercn/hotbuild/archive/refs/tags/v1.0.7.tar.gz"
+  sha256 "22d5feb6771ca1ba668b5371e0dccf06c4b201a41ac42a0651403e2798d68386"
+  license "GPL-3.0-only"
+  depends_on "go" => :build
+  def install
+    system "go", "build", *std_go_args
+  end
+  test do
```
Comment (carlocab) [inline Formula/hotbuild.rb:11]:
```suggestion
  license "GPL-3.0-only"

  depends_on "go" => :build

  def install
    system "go", "build", *std_go_args
  end

  test do
```

File: Formula/hotbuild.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    system "hotbuild"
```
Comment (carlocab) [inline Formula/hotbuild.rb:12]: Is there output here we can check?

**Review by carlocab — COMMENTED** [review]
We're not fans of self-submitted formulae, [unless they're very popular](https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff):

> We frown on authors submitting their own work unless it is very popular.

I don't think just barely enough stars to meet our notability requirement counts as "very popular".

I'm willing to see if another maintainer has a different opinion, however.

In the meantime, please rebase (with `git rebase`) and squash all your commits together so we can get rid of the `automerge skip` label.

**Comment by SMillerDev** [comment]
Seeing https://github.com/Homebrew/homebrew-core/pulls?q=is%3Apr+hotbuild+is%3Aclosed+

I'm gonna say it's not popular enough

**Comment by wandercn** [comment]
I don't think kn is popular enough. https://github.com/Homebrew/homebrew-core/pull/83053/commits . Why did it pass? @SMillerDev @carlocab 没有明确的标准是件很可怕的事情。难道开源社区也存在严重的双重标准。最可怕的是不按标准执行，全凭个人心情做事情。那我就再推广一段时间，请给一个明确的标准好吗？Would you please give a clear standard?

**Comment by SMillerDev** [comment]
> I don't think kn is popular enough.

It isn't a self-submission. It's a project from knative, the author of the pull request is not associated with knative. You wrote hotbuild and are trying to get it into homebrew by submitting a new pull request ever week despite being [warned](https://github.com/Homebrew/homebrew-core/pull/82161#issuecomment-889694442) not to do that anymore. The standard is this:

- It needs to be popular enough to pass our notability audits, and if the author submits their own software it needs to be significantly more popular than the audit requirements.

**Comment by wandercn** [comment]
Is it the author's push? It's not the same. For example, don't I just push with a new registered account or find a user to push for me? Do you think it's meaningful that the author's push leads to different standards and can easily cross such a distinction? 

算了这么麻烦，我已经对brew.sh失去兴趣了。我还是专心写代码吧。我不会再推送了。
Forget the trouble, I've lost interest in brew.sh. I'd better concentrate on writing code. I won't push anymore.

**Comment by SMillerDev** [comment]
> For example, don't I just push with a new registered account or find a user to push for me? Do you think it's meaningful that the author's push leads to different standards and can easily cross such a distinction?

I guess you can cheat any rule we have, but in general if a project is popular it's users will create a formula rather than it's author. When hotbuild becomes more popular one of your users might submit the formula and we can look if it meets the requirements again then.

**Pull request closed without merging**